4-auth/books/model-datastore.js

```diff
@@ -17,7 +17,7 @@ const Datastore = require('@google-cloud/datastore');
 const config = require('../config');
 
 const ds = Datastore({
-  projectId: config.get('GCLOUD_PROJECT'),
+  projectId: config.get('GOOGLE_CLOUD_PROJECT'),
```

Comment:
Why is the project ID a required config? 

The code is supposed to take it from the metadata server or from the GOOGLE_APPLICATION_CREDENTIAL context.